modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/display/context/SegmentsExperienceSelectorDisplayContext.java

```diff
@@ -14,11 +14,13 @@
 
 package com.liferay.segments.web.internal.display.context;
 
+import com.liferay.petra.string.StringPool;
 import com.liferay.portal.kernel.exception.PortalException;
 import com.liferay.portal.kernel.json.JSONArray;
 import com.liferay.portal.kernel.json.JSONFactoryUtil;
 import com.liferay.portal.kernel.json.JSONObject;
 import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.language.LanguageUtil;
```

Comment:
Better if we avoid use of LanguageUtil, instead we should use Language.

The reason for that is that LanguageUtil is not a static Util, it is a static wrapper of the Language class, so if we use LanguageUtil we are not declaring the Language depedency explicitly, so the tests will be hard to do :(.